modules/MarketAnalysis.py

```diff
@@ -158,6 +160,17 @@ def update_market_thread(self, cur, levels=None):
         while True:
             try:
                 raw_data = self.api.return_loan_orders(cur, levels)['offers']
+                if self.exchange.lower() == 'bitfinex' and  self.api.req_period > 1500:
```

Comment:
Why not reduce gradually?

Reply:
I did think of that, but I couldn't find a good place to trigger the reduction. The only way I could think of was a timer, which I didn't really want to implement. 

I actually thought this was fine, it's unlikely that it will actually be hit. I've only pushed it over 1500 when running loads of threads.